src/subsCache.coffee

```diff
@@ -130,15 +131,22 @@ class @SubsCache
               @delayedStop()
             #  catch err
             #    console.info 'Warning! SubsCache ignoring exception:', err.message
-          stop: -> @delayedStop()
+          stop: ->
+            if @count <= 0
+              @delayedStop()
+            else
+              @count--
           delayedStop: ->
             if expireTime >= 0
+              console.info("Beginning delayed stop:", this)
```

Comment:
I don't think these logs belong in this PR as they're mostly useful for debugging subscache.